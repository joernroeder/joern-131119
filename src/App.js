import React from 'react'
import './tailwind.build.css'

import { FilesProvider } from './store/FileStore'

import AppHeader from './components/AppHeader'
import FilesList from './components/FilesList'

function App() {
  return (
    <main>
      <FilesProvider>
        <AppHeader />

        <FilesList />
      </FilesProvider>
    </main>
  )
}

export default App
