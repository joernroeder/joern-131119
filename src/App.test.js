import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'

import App from './App'

test('renders without crashing', () => {
  const { queryByText, container } = render(<App />)

  //expect(queryByText('Testing')).toBeInTheDocument()
  expect(container).toMatchSnapshot()
})
