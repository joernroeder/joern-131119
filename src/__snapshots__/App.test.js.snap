// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div>
    <h1
      class="bg-black text-white"
    >
      Testing
    </h1>
  </div>
</div>
`;
